# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ScriptKind.h
# Commit: 7d5683180ac3
# Full Hash: 7d5683180ac3f9e7cc96d68dde27b8b48642a7ac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r a48aff5f2555 -r 7d5683180ac3 js/loader/ScriptKind.h
--- a/js/loader/ScriptKind.h	Thu Feb 17 15:57:52 2022 +0000
+++ b/js/loader/ScriptKind.h	Thu Feb 17 15:57:52 2022 +0000
@@ -4,15 +4,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef mozilla_dom_ScriptKind_h
-#define mozilla_dom_ScriptKind_h
+#ifndef js_loader_ScriptKind_h
+#define js_loader_ScriptKind_h
 
-namespace mozilla {
-namespace dom {
+namespace JS::loader {
 
 enum class ScriptKind { eClassic, eModule, eEvent };
 
-}  // namespace dom
-}  // namespace mozilla
+}  // namespace JS::loader
 
 #endif